src/compiler/utils/push_array.ts

```diff
@@ -0,0 +1,11 @@
+export function push_array(thisArray: any[], ...otherList: any[]) {
+  let count = 0;
+  for (let a = 0; a < otherList.length; a++) {
+    const other = otherList[a];
+    for (let i = 0; i < other.length; i++) {
+      thisArray.push(other[i]);
+    }
+    count += other.length;
+  }
+  return count;
```

Comment:
does this need to return `count`? the return value doesn't seem to ever be used

Reply:
compatibility with `Array.prototype.push`, almost zero cost